### PR TITLE
LPS-122028 create unit tests

### PR DESCRIPTION
**Pull request by marcosapmf**
Related Issue: https://issues.liferay.com/browse/LPS-122028

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js [hunk 1]
```diff
@@ -185,6 +185,7 @@ class Sidebar extends Component {
 							: false,
 					readOnly:
 						field.fieldName == 'name' ? readOnlyFieldName : false,
+					visible: field.fieldName !== 'name',
 				};
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js:189]: @marcosapmf dei deploy e minha sidebar ta tipo a imagem abaixo. Ta assim contigo também?

image

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js:189]: @carolmariaabb, não tava. Deixa eu olhar de novo

Comment (marcosapmf) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/Sidebar/Sidebar.es.js:189]: @carolmariaabb, corrigi :D

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @marcosapmf, can you please check if these failures are related to your changes? Thanks

**Comment by carolmariaabb** [comment]
@marcosapmf to corrigindo os testes de poshi que falharam.

**Comment by carolmariaabb** [comment]
Just started reviewing :)<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carolmariaabb** [comment]
Pull request submitted to [carolmariaabb#239](https://github.com/carolmariaabb/liferay-portal/pull/239).<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carolmariaabb** [comment]
Closed [#236](https://github.com/carolmariaabb/liferay-portal/pull/236).<br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**